src/GraphQL/Validation/TypeInfo.cs

```diff
@@ -34,7 +33,14 @@ public TypeInfo(ISchema schema)
         /// <returns></returns>
         public INode[] GetAncestors()
```

Comment:
Look at `GetAncestors()` usages. Only two places with index access. Let's refactor `GetAncestors` into `GetAncestor(int index)`. Also you can create `stackalloc` array instead of calling `ToArray` and then call `CopyTo`, so no array allocation 😄

Reply:
Sounds great!  Do you want to create PR?  Or do you want me to do so?

Reply:
I am busy right now, it will be faster for you to do it yourself.

Reply:
For now, an issue was posted

Reply:
ok